x-pack/plugins/actions/server/raw_action_schema.ts

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { schema } from '@kbn/config-schema';
+
+export const rawActionSchema = schema.object({
```

Comment:
should this actually be `rawConnectorSchema` and `rawPreconfiguredConnectorSchema`?

Reply:
I'll move this to task registry with the POC.
Let's compare this and the POC and decide :)